packages/steppp-react/package.json

```
@@ -0,0 +1,37 @@
+{
+    "name": "@ramseyinhouse/steppp-react",
+    "version": "0.0.1",
```

arguably a deployed major version, no?

```suggestion
    "version": "1.0.0",
```